skimage/feature/_texture.pyx

```diff
@@ -12,24 +12,38 @@ from .._shared.transform cimport integrate
 cdef extern from "numpy/npy_math.h":
     double NAN "NPY_NAN"
 
-
-def _glcm_loop(cnp.uint8_t[:, ::1] image, double[:] distances,
+ctypedef fused any_int:
```

Comment:
I just tested, and apparently this is not sufficient. I did not find information on whether unsigned are also part of python.integral.